<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateDictionaryDialog</class>
 <widget class="QDialog" name="GenerateDictionaryDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Générer un dictionnaire</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Fichier de sortie :</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtDictionaryFileName"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDictionaryFileName">
       <property name="text">
        <string>Parcourir...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CharsetSelector" name="charsetSelectorWidget" native="true"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblMaxPwdLenght">
       <property name="text">
        <string>Longueur max des combinaisons :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinMaxCombinationLength">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnGenerateDictionary">
       <property name="text">
        <string>Générer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseDialog">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CharsetSelector</class>
   <extends>QWidget</extends>
   <header>CharsetSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
